test/OpenTelemetry.Exporter.OpenTelemetryProtocol.Tests/OtlpRetryTests.cs

```diff
@@ -186,16 +188,6 @@ public static IEnumerable<object[]> GetGrpcTestCases()
                     },
                     expectedRetryAttempts: 9),
             };
-
-            yield return new[]
```

Comment:
@alanwest - I did not quite understand the significance of this test. It was testing that we would retry if the deadline is not set and server responds with a very high delay. We should not run in to this case as we would always have the deadline set, let me know if I am missing something here.